### PR TITLE
added map on guess fragment

### PR DESCRIPTION
**Pull request by JeremyDezalos**

### CODE REVIEW

File: app/src/main/java/com/github/wnder/guessLocation/GuessFragment.java [hunk 1]
```diff
@@ -0,0 +1,243 @@
...
+            startActivity(intent);
+        }
+        else{
+            AlertDialog alert = AlertBuilder.okAlert(getString(R.string.no_connection), getString(R.string.no_internet_guess), getActivity());
+            alert.show();
+        }
+    }
+
+    //Necessary overwrites for MapView lifecycle methods
+    /**
```
Comment (rkuenzi-epfl) [inline app/src/main/java/com/github/wnder/guessLocation/GuessFragment.java:130]: Just for clarity, I would move all the overrides that don't do anything to the end of the file an put actual code like `onProgressChanged` up there.

File: app/src/main/java/com/github/wnder/guessLocation/GuessFragment.java [hunk 2]
```diff
@@ -0,0 +1,243 @@
...
+        super.onDestroy();
+        mapView.onDestroy();
+    }
+
+    /**
+     * Manage the SeekBar
+     */
+    private void manageSeekBar(SeekBar radiusSeekBar, TextView radiusTextView){
+        //Set radius seekbar depending on user selected radius
+        int userRad = GlobalUser.getUser().getRadius();
```
Comment (rkuenzi-epfl) [inline app/src/main/java/com/github/wnder/guessLocation/GuessFragment.java:200]: you should use the existing user

File: app/src/main/java/com/github/wnder/guessLocation/GuessFragment.java [hunk 3]
```diff
@@ -0,0 +1,243 @@
...
+
+    /**
+     * When user interacts with radius seekbar
+     * @param seekBar radius seekbar
+     * @param progress step of seekbar
+     * @param fromUser boolean
+     */
+    @Override
+    public void onProgressChanged(SeekBar seekBar, int progress, boolean fromUser) {
+        User u = GlobalUser.getUser();
```
Comment (rkuenzi-epfl) [inline app/src/main/java/com/github/wnder/guessLocation/GuessFragment.java:219]: use existing user

**Review by rkuenzi-epfl — APPROVED** [review]
I only have really minor changes to propose so I'll approve so that you don't have to wait for me to review again just for these.

File: app/src/main/java/com/github/wnder/guessLocation/GuessFragment.java [hunk 4]
```diff
@@ -0,0 +1,243 @@
+package com.github.wnder.guessLocation;
+
+import android.content.Context;
```
Comment (amattei777) [inline app/src/main/java/com/github/wnder/guessLocation/GuessFragment.java:3]: You can remove import

File: app/src/main/java/com/github/wnder/guessLocation/GuessFragment.java [hunk 5]
```diff
@@ -0,0 +1,243 @@
...
+        radiusTextView = rootView.findViewById(R.id.radiusTextView);
+        manageSeekBar(radiusSeekBar, radiusTextView);
+
+        FloatingActionButton guessButton = rootView.findViewById(R.id.navigationToGuessButton);
+        guessButton.setOnClickListener((view) -> openPreviewActivity());
+
+        return rootView;
+    }
+
+    private void setZoom(){
```
Comment (amattei777) [inline app/src/main/java/com/github/wnder/guessLocation/GuessFragment.java:89]: You could have used the zoom class to have an animation on the zoom change. That being said it is more work and it work really well like that already so you don't need to change it for me to approve your PR.

**Review by amattei777 — CHANGES_REQUESTED** [review]
If the the radius map could be on top of the map maybe it would look better. Idk how much work it implies tho so it's fine as it is for now.

**Comment by JeremyDezalos** [comment]
> If the the radius map could be on top of the map maybe it would look better. Idk how much work it implies tho so it's fine as it is for now.

The radius map ???

**Comment by JeremyDezalos** [comment]
> If the the radius bar could be on top of the map maybe it would look better. Idk how much work it implies tho so it's fine as it is for now.

We decided that it will be done on Alois' PR since he already changed the theme of the bar on his side.

File: app/src/main/java/com/github/wnder/guessLocation/GuessFragment.java (hunk 2 above)
Comment (JeremyDezalos) [reply app/src/main/java/com/github/wnder/guessLocation/GuessFragment.java]: done

File: app/src/main/java/com/github/wnder/guessLocation/GuessFragment.java (hunk 3 above)
Comment (JeremyDezalos) [reply app/src/main/java/com/github/wnder/guessLocation/GuessFragment.java]: done

File: app/src/main/java/com/github/wnder/guessLocation/GuessFragment.java (hunk 1 above)
Comment (JeremyDezalos) [reply app/src/main/java/com/github/wnder/guessLocation/GuessFragment.java:154]: done

File: app/src/main/java/com/github/wnder/guessLocation/GuessFragment.java (hunk 4 above)
Comment (JeremyDezalos) [reply app/src/main/java/com/github/wnder/guessLocation/GuessFragment.java:3]: done

**Review by amattei777 — APPROVED** [review]

**Pull request merged**